example/integrations/mpi/mpi-example.yaml

```diff
@@ -23,7 +23,7 @@ spec:
                 - |
                   MPI_HOST=`cat /etc/volcano/mpiworker.host | tr "\n" ","`;
                   mkdir -p /var/run/sshd; /usr/sbin/sshd;
-                  mpiexec --allow-run-as-root --host ${MPI_HOST} -np 2 mpi_hello_world;
+                  mpiexec --allow-run-as-root --mca --host ${MPI_HOST} -np 2 mpi_hello_world;
```

Comment:
Does this parameter have any effect on ipv4?

Reply:
I'm not entirely sure. That's why I wanted someone to test it on ipv4. If we don't find someone to help, I can try spining up minikube, and test it there.

Reply:
I test in kind, with arm64 and ipv4, it works fine.But we'd better explain what this parameter is for

Reply:
> But we'd better explain what this parameter is for

That's a great question, and one that I was also curious about. I am actually new to learning MPI myself, and I have been experimenting with various methods. It turns out that using the "--mca" flag has been successful for me, but I did some further research and discovered that it actually requires a key value pair!

```
-mca, --mca <key> <value>
    Send arguments to various MCA modules. See the "MCA" section, below. 
```

https://www.open-mpi.org/doc/v2.1/man1/mpiexec.1.php

I'm confused now.  Hope some expert in OpenMPI can provide clarification and help me understand it better. I can also dig more tomorrow. Time for bed now. Thanks for the help!

Reply:
I retested using 22.04, and it appears to be working without the need for the --mca option. 

```
$ grep mpiexec mpi-example.yaml
                  mpiexec --allow-run-as-root --host ${MPI_HOST} -np 2 mpi_hello_world; sleep 600;

$ k -n gengwg logs  lm-mpi-job-mpimaster-0
Warning: Permanently added 'lm-mpi-job-mpiworker-0.lm-mpi-job' (ED25519) to the list of known hosts.
Warning: Permanently added 'lm-mpi-job-mpiworker-1.lm-mpi-job' (ED25519) to the list of known hosts.
Hello world from processor lm-mpi-job-mpiworker-0, rank 0 out of 2 processors

$ k -n gengwg exec -it lm-mpi-job-mpimaster-0 -- mpirun --version
mpirun (Open MPI) 4.1.2
```

Let me update this PR to remove the --mca. Image/MPI version update seems sufficient.

Reply:
Done.  Maybe someone could rebuild and upload the image to Docker Hub, such as example-mpi:0.0.3, if it is not built automatically. It would be beneficial for others experiencing the same issue.

Reply:
After the pr is merged, I can push the new example-mpi image.

Reply:
> After the pr is merged, I can push the new example-mpi image.

please also modify https://github.com/volcano-sh/volcano/blob/9d0a5daadb663a1e9737386965132f1e76fb9993/example/integrations/mpi/mpi-example.yaml#L27

Reply:
yes, you are right, thanks!